.github/workflows/vast.yaml

```diff
@@ -884,7 +897,6 @@ jobs:
       HOMEBREW_NO_ANALYTICS: 1
       HOMEBREW_NO_INSTALL_CLEANUP: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
-      DESTDIR: "${{ github.workspace }}"
```

Comment:
Note to the reviewer: Setting DESTDIR and CMAKE_INSTALL_PREFIX together breaks installations. This wasn't ever used to begin with.